.github/workflows/coverity.yml

```diff
@@ -35,11 +35,11 @@ jobs:
 
     - name: Download Coverity
       run: |
```

Comment:
You could use `>` instead for a multiline command. This would increase readability even more.